### DISCUSSION TITLE
Targeting Issues on AC

### DISCUSSION THREAD

**Discussion by Kyroth88** [trickerer/Trinity-Bots#820] [Q&A]
Hello

I have modified the config file to have dps priority as skull and tank priority as cross, however the bots are not following my commands, which is particularly frustrating in raids where I need to dps adds equally.

I have a feeling this is user error, and not an issue with the module so I was looking for some suggestions on resolving this.

Again all I have done is
1) modified config so dps = skill tank = cross
2) restarted world server
3) mark targets with cross and with skull in raid

Thank you very much for helping out on this.

**Comment by trickerer** [comment]
The only thing which comes to mind is encounter-specific targets with targeting priority above raid icons. Otherwise priority goes from skull to star for each role (tanking is higher prio for Tank+DPS bots) so if you set skull for DPS and cross for Tanking all tanks go for cross and all DPS go for skull. I personally use skull for DPS and skull+cross for Tanking in case I need bots to tank 2 targets at once. I tested separate icons: skull for DPS and cross for Tanking, and it works as intended.

**Comment by Kyroth88** [comment]
Thank you very much! I decided to clean house on my build, I had some other issues in my database so the fresh build with the latest release seemed to fix it!

I like your idea of the skull and cross mask for tanking! Do you have any other tips or tricks you have come up with on targeting masks when you play?

Thank you so much for this, I cannot tell you how much joy being able to do these raids again on my own has brought me!
